src/emulator/apphosting/serve.ts

```diff
@@ -167,3 +201,44 @@ export function getEmulatorEnvs(): Record<string, string> {
 
   return envs;
 }
+
+async function tripFirebasePostinstall(
+  rootDirectory: string,
+  env: Record<string, string>,
+): Promise<void> {
+  let firebaseUtilPath;
+  try {
+    firebaseUtilPath = dirname(
+      require.resolve("@firebase/util/package.json", { paths: [rootDirectory] }),
```

Comment:
If for whatever reason there is more than one version of `@firebase/util` this is non-deterministic on which one it patches. Should patch all of them.